### DISCUSSION TITLE
Fastest Route Navigation Plugin

### DISCUSSION THREAD

**Discussion by Crrimsons** [runelite/runelite#18093] [Ideas]
I'm playing osrs for years and with time I keep finding there are much quicker ways to get to so many places that I used to take pretty slow roads, this is especially usefull when doing quests and for ironmans in general.
 
The idea is simple:
A players would be able to mark a location on the world map and receive efficient routes(or atleast a fast one), from his location, considering available travel methods like teleports, fairy rings, charter ships, etc, and prioritizing methods unlocked for the player's account based on level requirements.

**Comment by YvesW** [comment]
https://runelite.net/plugin-hub/show/shortest-path
